client/components/CustomBankField.js

```diff
@@ -0,0 +1,66 @@
+import {has, translate as t} from '../Helpers';
+import T from './Translated';
+
+export default class CustomBankField extends React.Component {
+
+    constructor(props) {
+        has(props, 'params');
+
+        super(props);
+    }
+
+    getValue() {
+        let node = this.refs[this.props.params.name].getDOMNode();
+        return {
+            name: this.props.params.name,
+            value: (this.props.params.type === "number") ? parseInt(node.value, 10) : node.value
+        };
+    }
+
+    setValue(value) {
```

Comment:
The only use of `setValue` does `setValue("")`, which won't reset the select input, as `""` has likely no chances to be a valid option in the field. How about dropping this method? When reopening the modal, the previous value will be still selected, but that's not much a big deal.

About that topic: when you open the modal, I don't think the previous set value will be selected by default, which can be confusing to the user. For instance, if I've set my birthday to 01011970 when creating my account the first time, I don't think this will be shown when I open the modal. Is this correct, or did I miss something?